guides/schema/mutations.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Schema — Mutations
+---
+
+## Mutations
+
+Registering a mutation root allows to define fields that can mutate your data.
+
+```ruby
+Schema = GraphQL::Schema.define do
+  query QueryRoot
+  mutation MutationRoot
+end
+
+MutationRoot = GraphQL::ObjectType.define do
+  name "Mutation"
+
+  field :addPost, Post do
+    description "Adds a Post."
+
+    # Use Input Types to define complex argument types
+    argument :post, PostInputType
+    resolve ->(t, args, c) {
+      title = args['post']['title']
+      description = args['post']['description']
+      Post.create(title: title, description: description)
+    }
+  end
+end
+
+PostInputType = GraphQL::InputObjectType.define do
+  name "PostInputType"
+  description "Properties for creating a Post"
+
+  argument :title, !types.String do
+    description "Title of the post."
+  end
+
+  argument :description, types.String do
+    description "Description of the post."
+  end
+end
+```
+
+## Nested mutations
```

Comment:
The use case would be that whenever creating items in bulk this really helps.
An added benefit of this is that it allows you to to easily wrap it in an transaction, when creating multiple records, making a mutation transaction safe. Which is for what i was using it in the first place. 

Is that something that we should maybe mention? I.e. with ruby sample code? I know graphql is db agnostic, but still graphql/activerecord is a common use case.